src/reducers/index.js

```diff
@@ -50,4 +51,5 @@ export default combineReducers({
   breach: BreachReducer,
   rowDetail: RowDetailReducer,
   permission: PermissionReducer,
+  vaccine: VaccineReducer,
```

Comment:
I kind of have a feeling we might want this named something more appropriate to bluetooth but if we do we can change it then heh